chromium_src/chrome/browser/ui/autofill/payments/save_card_bubble_controller_impl.cc

```diff
@@ -3,10 +3,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#include "../../../../../../../chrome/browser/ui/autofill/payments/save_card_bubble_controller_impl.h"
-
-#define BRAVE_SHOULD_SHOW_SIGNIN_PROMO_SHORT_CIRCUIT_TO_FALSE \
-return false;
+#define CHECK_SYNC_POLICY \
+  if (sync_service && sync_service->HasDisableReason( \
```

Comment:
I'm confused, if we're still going to patch here then why not just keep the original patch? I was trying to avoid patching here by using the DisableReason, but actually I think enabling kAutofillDoNotUploadSaveUnsupportedCards by default might work

Reply:
DisableReason will affect our sync function.
`kAutofillDoNotUploadSaveUnsupportedCards` should work, I will change it

Reply:
https://github.com/brave/brave-browser/issues/5728 follow up issued created